backward.hpp

```diff
@@ -786,7 +786,7 @@ class TraceResolverLinuxImpl<trace_resolver_tag::libbfd>:
 			if(len == -1) {
```

Comment:
`len <= -1`

Makes me feel safer ;)

Reply:
I will change this. Keep in mind I didn't modify this - only the parts shown with +/- (two lines total) have been modified.